treeherder/log_parser/tasks.py

```diff
@@ -77,23 +75,6 @@ def parse_logs(job_id, job_log_ids, priority):
     if first_exception:
         raise first_exception
 
-    if "errorsummary_json" in completed_names and "live_backing_log" in completed_names:
```

Comment:
This removal will leave some dead code.  I'll investigate and give some links to code to remove with this.

Reply:
On second thought, I think I should just bite the bullet and create a separate PR to remove this `autoclassify` and `crossreference` code.  That way it won't gum up your PR and we could more easily resurrect it later if we ever decide to.  Let's leave your changes as-is and I'll remove the dead code in a separate PR.  I'll have a few things to check to make sure we don't break anything.

Reply:
Ok! I think I've addressed all your review comments, then. Should we bake https://github.com/mozilla/treeherder/pull/6891/commits/59700923c150ca197ea9520e0ef1d1b0cba38cdf in prototype?

Reply:
Yep, I'll push your new code to prototype now.  :)